Library/Homebrew/cmd/help.rb

```diff
@@ -21,6 +20,7 @@
 
 Further help:
   man brew
+  brew help COMMAND
```

Comment:
My thinking was that this is the output of `brew help`, thus executing a plain `brew help` won't actually provide any “further help” (as the section title claims).